textattack/models/wrappers/pytorch_model_wrapper.py

```diff
@@ -8,14 +9,15 @@
 class PyTorchModelWrapper(ModelWrapper):
     """Loads a PyTorch model (`nn.Module`) and tokenizer."""
 
-    def __init__(self, model, tokenizer, batch_size=32):
+    def __init__(self, model, tokenizer, loss=CrossEntropyLoss(), batch_size=32):
```

Comment:
Can you add `loss` to the docstring? (This docstring was already kind of incomplete, sorry)